app/models/apps/ep_vote_app/application_form.rb

```diff
@@ -2,6 +2,7 @@ module Apps
   module EpVoteApp
     class ApplicationForm
       DELIVERY_BY_POST_DEADLINE_DATE = Date.new(2019, 5, 3)
+      INACTIVE = true
```

Comment:
Toto je trochu divne, asi by som tam dal datum alebo to otocil na `ACTIVE`

Reply:
Zmenil som to na class metodu `active?` :)